pkg/apis/aquasecurity/v1alpha1/config_audit_types.go

```diff
@@ -70,6 +70,7 @@ type ConfigAuditReportList struct {
 // TODO my-node)
 type ConfigAudit struct {
 	Scanner         Scanner            `json:"scanner"`
+	LastUpdated     LastUpdated        `json:"lastUpdated"`
```

Comment:
In `ObjectMata` there's the `CreationTimestamp` field so maybe we should call this one `UpdateTimestamp`. I don't have strong opinions on that. I just like consistency.